ansible/roles/csm.rr.ceph_zoning/README.md

```diff
@@ -0,0 +1,37 @@
+csm.sbps.lio_config
```

Comment:
This line, and the rest of the contents of the file, do not appear to be correct for this role